### PR TITLE
complete site

### PR DESCRIPTION
**Pull request by juliusb0205**
@ming-ancino

### CODE REVIEW

**Comment by ming-ancino** [comment]
@kencruz07

File: app/assets/stylesheets/static_pages.css.scss [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  position: absolute;
+  top: 250px;
+}
+
+.reservations {
+  position: absolute;
+  top: 300px;
+  left: 230px;
+  width: 450px;
+  float: left
```
Comment (kencruz07) [inline app/assets/stylesheets/static_pages.css.scss]: Missing ';' 😉

File: app/assets/stylesheets/static_pages.css.scss [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+}
+
+h4 {
+  color: white;
+  font-family: "VerlagLight", CenturyGothic, AppleGothic, sans-serif;
+  font-size: 30px;
+  font-weight: lighter;
+  letter-spacing: 0.1em;
```
Comment (kencruz07) [inline app/assets/stylesheets/static_pages.css.scss]: Usually, we remove the `0` when using decimal values `i.e. .1em`

File: app/assets/stylesheets/static_pages.css.scss [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    color: white;
+    text-decoration: underline;
+    font-size: 14px;
+    margin-right: 20px;
+  }
+}
+
+input,
+input::-webkit-input-placeholder {
+    font-size: 16px;
```
Comment (kencruz07) [inline app/assets/stylesheets/static_pages.css.scss]: Use only 2 spaces for this block. 🖖🏼

File: app/assets/stylesheets/static_pages.css.scss [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    font-weigth: 300;
+    line-height: 2;
+    padding: 5px 10px;
+    letter-spacing: 0.1em;
```
Comment (kencruz07) [inline app/assets/stylesheets/static_pages.css.scss]: Same as above about decimals.

File: app/assets/stylesheets/static_pages.css.scss [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+.social-media {
+  margin-bottom: 5px;
+}
+
+.submit {
+  width: 105px;
+  height: 51px;
+}
+
+input[type="submit"] {
```
Comment (kencruz07) [inline app/assets/stylesheets/static_pages.css.scss]: I'm not sure about this, but can we use single quotes instead?

File: app/assets/stylesheets/static_pages.css.scss [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+    -webkit-appearance: button;
+    cursor: pointer;
+    color: #f4a73a;
+    height: 51px;
+    padding: 0px;
+    font-weight: bold;
+}
+
+input[type="submit"]:hover {
```
Comment (kencruz07) [inline app/assets/stylesheets/static_pages.css.scss]: Same as above.

File: app/assets/stylesheets/static_pages.css.scss [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+
+h4 {
+  color: white;
+  font-family: "VerlagLight", CenturyGothic, AppleGothic, sans-serif;
+  font-size: 30px;
+  font-weight: lighter;
+  letter-spacing: 0.1em;
+  margin-bottom: 10px;
+}
+
```
Comment (kencruz07) [inline app/assets/stylesheets/static_pages.css.scss]: Remove extra line break. 👍🏼

File: app/assets/stylesheets/static_pages.css.scss (hunk 5 above)
Comment (kencruz07) [inline app/assets/stylesheets/static_pages.css.scss]: Also, 2 spaces for this block.

File: app/assets/stylesheets/static_pages.css.scss (hunk 6 above)
Comment (kencruz07) [inline app/assets/stylesheets/static_pages.css.scss]: Same as above about spacing.

**Comment by kencruz07** [comment]
@ming-ancino Please check. Thanks thanks! 👍

**Comment by ming-ancino** [comment]
@juliusb0205 could you apply this http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/
Discuss pros and cons afterwards.

**Comment by juliusb0205** [comment]
@ming-ancino 

updated lines that were commented on by ken
implement BEM

**Pull request merged**